src/Npgsql.NodaTime/IntervalHandler.cs

```diff
@@ -10,26 +11,19 @@ public class IntervalHandlerFactory : NpgsqlTypeHandlerFactory<Period>
     {
         // Check for the legacy floating point timestamps feature
         protected override NpgsqlTypeHandler<Period> Create(NpgsqlConnection conn)
-            => new IntervalHandler(conn.HasIntegerDateTimes);
+        {
```

Comment:
nit: use express-bodied method with conditional operator (and in other instances below).

Reply:
Ah, meant to convert before pushing. Will fix.